python/tvm/contrib/xcode.py

```diff
@@ -126,8 +126,13 @@ def compile_metal(code, path_target=None, sdk="macosx"):
         out_file.write(code)
     file_target = path_target if path_target else temp_target
 
+    # See:
+    # - https://developer.apple.com/documentation/metal/gpu_functions_libraries/building_a_library_with_metal_s_command-line_tools#overview
```

Comment:
> ************* Module tvm.contrib.xcode
> C:130, 0: Line too long (139/100) (line-too-long)

Can we suppress this warning? I think having the URL here is important.

Reply:
sure, add a pylint suppression.

Reply:
Looks good now.